jobserver/models.py

```diff
@@ -321,6 +321,55 @@ def status(self):
         return "unknown"
 
 
+class Membership(models.Model):
+    project = models.ForeignKey(
+        "Project",
+        on_delete=models.CASCADE,
+        related_name="members",
+    )
+    user = models.ForeignKey(
+        "User",
+        on_delete=models.CASCADE,
+        related_name="memberships",
+    )
+
+    class Meta:
+        unique_together = ["project", "user"]
+
+    def __str__(self):
+        return f"{self.user.username} member of {self.project.name}"
+
+
+class Org(models.Model):
+    name = models.TextField(unique=True)
+
+    class Meta:
+        verbose_name = "Organisation"
+
+    def __str__(self):
+        return self.name
+
+
+class Project(models.Model):
+    org = models.ForeignKey(
+        "Org",
+        on_delete=models.CASCADE,
+        related_name="projects",
+    )
+
+    name = models.TextField(unique=True)
+    email = models.TextField()
+    project_lead = models.TextField()
+    proposed_start_date = models.DateTimeField()
+    proposed_duration = models.TextField()
```

Comment:
Don't know enough about the requirements -- but a `proposed_end_date` would allow you to do more reporting.

Reply:
This is all a bit loose currently, the data is very free form so I'm beinf very relaxed about fields like this until we get a better sense of what's needed from use.